poetry/packages/vcs_dependency.py

```diff
@@ -21,7 +29,7 @@ def __init__(
         self._rev = rev
 
         super(VCSDependency, self).__init__(
-            name, "*", optional=optional, allows_prereleases=True
+            name, "*", optional=optional, **kwargs, allows_prereleases=True
```

Comment:
`**kwargs` should be the last one.

Reply:
Force push, or append a commit to change this?

Reply:
It's up to you and I'm wondering where would this thread goes after force push.